js/everything.js

```diff
@@ -490,13 +508,17 @@ $(function() {
     // Handle form submission
     $("#form").submit(function(event) {
         event.preventDefault();
-
         var formValue = $.trim($("#v").val());
+        var formValueTime = formValue.match(/(&t=\d*)$/g);
```

Comment:
This gathers the whole string `&t=X` which I put straight into the `makeListenURL` function.